teuthology/packaging.py

```diff
@@ -422,6 +424,30 @@ def _get_config_value_for_remote(ctx, remote, config, key):
     return config.get(key)
 
 
+def _get_response(url, wait=False, sleep=15, tries=10):
+    with safe_while(sleep=sleep, tries=tries, _raise=False) as proceed:
+        while proceed():
+            resp = requests.get(url)
+            if resp.ok:
+                log.info('Package found...')
+                break
+
+            if not wait:
+                log.info(
+                    'Package is not found at: %s (got HTTP code %s)...',
+                    url,
+                    resp.status_code,
+                )
+                break
+
+            log.info(
+                'Package not there yet (got HTTP code %s), waiting...',
```

Comment:
if `wait` is True and we never are able to retrieve the package it would be useful to know the `url` here.